realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -79,6 +79,19 @@
  * A standard pattern for working with Realm in Android activities can be seen below:
  * <p>
  * <pre>
+ * public class RealmApplication extends Application {
+ *
+ *     private static RealmConfiguration defaultRealmConfig;
```

Comment:
uh... i used to have 
```
if (defaultRealmConfig == null) {
    defaultRealmConfig = new RealmConfiguration.Builder(this).build();
}
```
in `onCreate`. But then i think it might confuse user, so it is removed. Forgot to change the static.

What do you think? Does init only once would create some value for beginners?